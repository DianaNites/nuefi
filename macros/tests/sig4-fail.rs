struct Embedded {
    //
}

impl Embedded {
    #[macros::entry]
    fn e_main(&self, table: ()) {}
}

fn main() {}
