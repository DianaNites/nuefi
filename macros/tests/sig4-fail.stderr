error: cannot be a method
 --> tests/sig4-fail.rs:7:15
  |
7 |     fn e_main(&self, table: ()) {}
  |               ^^^^^
